examples/ipp_aes_ctr_example.cc

```diff
@@ -0,0 +1,70 @@
+//  Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
+//  Copyright (c) 2020 Intel Corporation
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+
+#include <cstdio>
+#include <string>
+
+#include "rocksdb/db.h"
+#include "rocksdb/env_encryption.h"
+#include "rocksdb/options.h"
+#include "rocksdb/slice.h"
+#include "rocksdb/utilities/options_util.h"
+
+using namespace ROCKSDB_NAMESPACE;
+
+std::string kDBPath = "/tmp/ipp_aes_example";
+
+int main() {
+  setbuf(stdout, NULL);
+
+#if defined(IPPCP) && (defined(HAVE_SSE42) || defined(HAVE_SSE2))
+  DB* db;
```

Comment:
You probably do not need this ifdef and can use the return from EncryptoinProvider::CreateFromString?

Reply:
The ifdef is added to avoid a fail on the unit tests on systems with no ipp-crypto - e.g. the CircleCI checks would fail on the assert of the status returned from EncryptionProvider::CreateFromString (line 31).